catalogue/webapp/pages/search/works.tsx

```diff
@@ -93,17 +93,24 @@ export const CatalogueSearchPage: NextPageWithLayout<Props> = ({
     router.push({ pathname: router.pathname, query: newQuery });
   }, [sortOrder]);
 
-  if (works.totalResults === 0) return <p>nothing</p>;
```

Comment:
This will get handled differently (through `SearchNoResults`, as there are two "no results" to consider;
- No results with a query
- No query